server/server.js

```diff
@@ -1,6 +1,15 @@
 const express = require("express");
+const mongoose = require("mongoose");
 const app = express();
+const { dbPassword } = require("./config");
 const port = 3000;
+console.log(dbPassword);
+const URI = `mongodb://user1:${dbPassword}@ds111455.mlab.com:11455/v8-chingu-geckos-15`;
```

Comment:
It is best practice to use environment variables here, with `dotenv` npm package: https://www.npmjs.com/package/dotenv

Basically you'll have a `.env` file (added in gitignore), and you declare the secret variables inside it: `DB_PASSWORD=...` or the entire DB_URL